src/populate-board.js

```diff
@@ -0,0 +1,16 @@
+const scoreList = document.getElementById('score-list');
```

Comment:
It would be a cleaner implementation to pass this into the function. So if the HTML ever changes, the index.js can control which element this function will manipulate.

Reply:
Okay cool